Account Onboard Utility/Accounts_Onboard_Utility.ps1

```diff
@@ -291,10 +291,12 @@ Function New-AccountObject
 		$_Account.secretManagement.automaticManagementEnabled = Convert-ToBool $AccountLine.enableAutoMgmt
 		if ($_Account.secretManagement.automaticManagementEnabled -eq $false)
 		{ $_Account.secretManagement.manualManagementReason = $AccountLine.manualMgmtReason }
-		$_Account.remoteMachinesAccess = "" | select "remoteMachines", "accessRestrictedToRemoteMachines"
-		$_Account.remoteMachinesAccess.remoteMachines = $AccountLine.remoteMachineAddresses
-		$_Account.remoteMachinesAccess.accessRestrictedToRemoteMachines = Convert-ToBool $AccountLine.restrictMachineAccessToList
-		
+		If($AccountLine.remotemachines -ne $Null -or "")
```

Comment:
Better to use:
[String]::IsNullOrEmpty()
So:
If(-not [String]::IsNullOrEmpty($AccountLine.remoteMachines))
{}

Reply:
Thanks, I was looking at '[String]::IsNullOrEmpty()' but I could not figure out how to set this to 'not'.